javaharness/java/arcs/android/AndroidPortableJson.java

```diff
@@ -9,15 +9,15 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
-class PortableJsonAndroidImpl implements PortableJson {
+public class AndroidPortableJson implements PortableJson {
```

Comment:
this is one of the examples of what i mean by suggesting to break this PR into smaller PRs. 
I don't mind either of the names, I'm happy with the new AndroidPortableJson name instead of the old longer one. But the rename could easily be a separate PR. 

I'm happy to do a series of quick iterations on small PRs, I understand that everyone is blocked by this work. But even just having cosmetic changes like renames in a separate change makes it much easier (or even possible) to actually significantly understand and  comment on significant changes.

Reply:
Good point. I made another PR just renaming these classes so they fit the same style. Sorry to make things so large as I discovered that there are some architectural changes I need to make during the refactoring and PRs kept coming in.